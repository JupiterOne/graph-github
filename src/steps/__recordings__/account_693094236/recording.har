{
  "log": {
    "_recordingName": "account",
    "creator": {
      "comment": "persister:JupiterOneIntegationFSPersister",
      "name": "Polly.JS",
      "version": "5.1.1"
    },
    "entries": [
      {
        "_id": "7f2926ac1ad814d2dfcaf9525a63bf36",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.machine-man-preview+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.11.4 octokit-core.js/3.5.1 Node.js/14.17.3 (linux; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "content-length",
              "value": "0"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 812,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "queryString": [],
          "url": "https://api.github.com/app/installations/17214088/access_tokens"
        },
        "response": {
          "bodySize": 348,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 348,
            "text": "{\"token\":\"[REDACTED]\",\"expires_at\":\"2050-12-31T18:09:20Z\",\"permissions\":{\"members\":\"read\",\"organization_administration\":\"read\",\"organization_secrets\":\"read\",\"actions\":\"read\",\"contents\":\"read\",\"environments\":\"read\",\"issues\":\"read\",\"metadata\":\"read\",\"pull_requests\":\"read\",\"secrets\":\"read\"},\"repository_selection\":\"all\"}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Tue, 01 Feb 2022 15:51:13 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "348"
            },
            {
              "name": "cache-control",
              "value": "public, max-age=60, s-maxage=60"
            },
            {
              "name": "vary",
              "value": "Accept, Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "etag",
              "value": "\"e30985bdc1151cb719e606a16f967f69b85864aaac965a3ac5a8bb80997c787a\""
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; param=machine-man-preview; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "x-github-request-id",
              "value": "F502:48E2:1655E7:32DE47:61F956F0"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1071,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 201,
          "statusText": "Created"
        },
        "startedDateTime": "2022-02-01T15:51:12.571Z",
        "time": 842,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 842
        }
      },
      {
        "_id": "90e92246ca868687b35a21f1736a0e7d",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.v3+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.11.4 octokit-core.js/3.5.1 Node.js/14.17.3 (linux; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 761,
          "httpVersion": "HTTP/1.1",
          "method": "GET",
          "queryString": [],
          "url": "https://api.github.com/app/installations/17214088"
        },
        "response": {
          "bodySize": 1363,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 1363,
            "text": "{\"id\":17214088,\"account\":{\"login\":\"Kei-Institute\",\"id\":84923503,\"node_id\":\"MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz\",\"avatar_url\":\"https://avatars.githubusercontent.com/u/84923503?v=4\",\"gravatar_id\":\"\",\"url\":\"https://api.github.com/users/Kei-Institute\",\"html_url\":\"https://github.com/Kei-Institute\",\"followers_url\":\"https://api.github.com/users/Kei-Institute/followers\",\"following_url\":\"https://api.github.com/users/Kei-Institute/following{/other_user}\",\"gists_url\":\"https://api.github.com/users/Kei-Institute/gists{/gist_id}\",\"starred_url\":\"https://api.github.com/users/Kei-Institute/starred{/owner}{/repo}\",\"subscriptions_url\":\"https://api.github.com/users/Kei-Institute/subscriptions\",\"organizations_url\":\"https://api.github.com/users/Kei-Institute/orgs\",\"repos_url\":\"https://api.github.com/users/Kei-Institute/repos\",\"events_url\":\"https://api.github.com/users/Kei-Institute/events{/privacy}\",\"received_events_url\":\"https://api.github.com/users/Kei-Institute/received_events\",\"type\":\"Organization\",\"site_admin\":false},\"repository_selection\":\"all\",\"access_tokens_url\":\"https://api.github.com/app/installations/17214088/access_tokens\",\"repositories_url\":\"https://api.github.com/installation/repositories\",\"html_url\":\"https://github.com/organizations/Kei-Institute/settings/installations/17214088\",\"app_id\":117431,\"app_slug\":\"jupiterone-kei-institute\",\"target_id\":84923503,\"target_type\":\"Organization\",\"permissions\":{\"issues\":\"read\",\"actions\":\"read\",\"members\":\"read\",\"secrets\":\"read\",\"contents\":\"read\",\"metadata\":\"read\",\"environments\":\"read\",\"pull_requests\":\"read\",\"organization_secrets\":\"read\",\"organization_administration\":\"read\"},\"events\":[],\"created_at\":\"2021-05-27T15:42:53.000Z\",\"updated_at\":\"2022-01-11T17:09:24.000Z\",\"single_file_name\":null,\"has_multiple_single_files\":false,\"single_file_paths\":[],\"suspended_by\":null,\"suspended_at\":null}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Tue, 01 Feb 2022 15:51:13 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "cache-control",
              "value": "public, max-age=60, s-maxage=60"
            },
            {
              "name": "vary",
              "value": "Accept, Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "etag",
              "value": "W/\"4662b5a4aa4ae852addefe7cfe4b37830f617deae0434f60ddc8e5bc9a704433\""
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "x-github-request-id",
              "value": "F503:7F2E:73B89:146B86:61F956F1"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1076,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 200,
          "statusText": "OK"
        },
        "startedDateTime": "2022-02-01T15:51:13.451Z",
        "time": 633,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 633
        }
      },
      {
        "_id": "22e4c882794d943e8a6f0961702d5000",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 506,
          "cookies": [],
          "headers": [
            {
              "name": "content-type",
              "value": "application/json"
            },
            {
              "name": "accept",
              "value": "application/json"
            },
            {
              "name": "user-agent",
              "value": "jupiterone-graph-github"
            },
            {
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 228,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "postData": {
            "mimeType": "application/json",
            "params": [],
            "text": "{\"query\":\"query ($login: String!) {\\n    organization(login: $login) {\\n        id\\n        ... organizationFields\\n      }\\n... rateLimit\\n  }\\n\\nfragment organizationFields on Organization {\\n    login\\n    name\\n    createdAt\\n    updatedAt\\n    description\\n    email\\n    databaseId\\n    isVerified\\n    location\\n    websiteUrl\\n    url\\n  }\\n\\nfragment rateLimit on Query {\\n    rateLimit {\\n      limit\\n      cost\\n      remaining\\n      resetAt\\n    }\\n  }\",\"variables\":{\"login\":\"Kei-Institute\"}}"
          },
          "queryString": [],
          "url": "https://api.github.com/graphql"
        },
        "response": {
          "bodySize": 481,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 481,
            "text": "{\"data\":{\"organization\":{\"id\":\"MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz\",\"login\":\"Kei-Institute\",\"name\":\"Kei-Institute\",\"createdAt\":\"2021-05-27T15:21:12Z\",\"updatedAt\":\"2022-01-06T17:48:33Z\",\"description\":\"Here's my test description\",\"email\":\"email@email.com\",\"databaseId\":84923503,\"isVerified\":false,\"location\":\"Albania\",\"websiteUrl\":\"www.goclickatesturl.com\",\"url\":\"https://github.com/Kei-Institute\"},\"rateLimit\":{\"limit\":5000,\"cost\":1,\"remaining\":4999,\"resetAt\":\"2022-02-01T16:51:14Z\"}}}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Tue, 01 Feb 2022 15:51:14 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "481"
            },
            {
              "name": "x-github-media-type",
              "value": "github.v4"
            },
            {
              "name": "x-ratelimit-limit",
              "value": "5000"
            },
            {
              "name": "x-ratelimit-remaining",
              "value": "4999"
            },
            {
              "name": "x-ratelimit-reset",
              "value": "1643734274"
            },
            {
              "name": "x-ratelimit-used",
              "value": "1"
            },
            {
              "name": "x-ratelimit-resource",
              "value": "graphql"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "vary",
              "value": "Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "x-github-request-id",
              "value": "F504:7937:45033:EEEA6:61F956F2"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1036,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 200,
          "statusText": "OK"
        },
        "startedDateTime": "2022-02-01T15:51:14.112Z",
        "time": 907,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 907
        }
      }
    ],
    "pages": [],
    "version": "1.2"
  }
}
