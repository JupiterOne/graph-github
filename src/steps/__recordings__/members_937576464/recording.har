{
  "log": {
    "_recordingName": "members",
    "creator": {
      "comment": "persister:JupiterOneIntegationFSPersister",
      "name": "Polly.JS",
      "version": "5.1.1"
    },
    "entries": [
      {
        "_id": "06d6593fc4a07c98683e9f0972b1c5e0",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.machine-man-preview+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.12.0 octokit-core.js/3.5.1 Node.js/16.14.2 (darwin; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "[REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "content-length",
              "value": "0"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 813,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "queryString": [],
          "url": "https://api.github.com/app/installations/23522163/access_tokens"
        },
        "response": {
          "bodySize": 102,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 102,
            "text": "{\"message\":\"A JSON web token could not be decoded\",\"documentation_url\":\"https://docs.github.com/rest\",\"token\":\"[REDACTED]\",\"expires_at\":\"2050-12-31T18:09:20Z\"}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 02 Sep 2022 15:59:41 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "102"
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; param=machine-man-preview; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "vary",
              "value": "Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "x-github-request-id",
              "value": "F4B9:689B:57D06:D196E:6312286D"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 939,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 401,
          "statusText": "Unauthorized"
        },
        "startedDateTime": "2022-09-02T15:59:40.909Z",
        "time": 160,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 160
        }
      }
    ],
    "pages": [],
    "version": "1.2"
  }
}
