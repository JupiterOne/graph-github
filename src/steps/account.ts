import {
  IntegrationStep,
  IntegrationStepExecutionContext,
} from '@jupiterone/integration-sdk-core';

import { IntegrationConfig } from '../config';
import { createAPIClient } from '../client';
import { toAccountEntity } from '../sync/converters';
import {
  GITHUB_ACCOUNT_ENTITY_TYPE,
  GITHUB_ACCOUNT_ENTITY_CLASS,
} from '../constants';

export const DATA_ACCOUNT_ENTITY = 'DATA_ACCOUNT_ENTITY';

export async function fetchAccountDetails({
  jobState,
  instance,
  logger,
}: IntegrationStepExecutionContext<IntegrationConfig>) {
  const config = instance.config;
  const apiClient = createAPIClient(config, logger);
  const accountEntity = await jobState.addEntity(
    toAccountEntity(await apiClient.getAccountDetails()),
  );

  await jobState.setData(DATA_ACCOUNT_ENTITY, accountEntity);
}

export const accountSteps: IntegrationStep<IntegrationConfig>[] = [
  {
    id: 'fetch-account',
    name: 'Fetch Account Details',
    entities: [
      {
        resourceName: 'Account',
        _type: GITHUB_ACCOUNT_ENTITY_TYPE,
        _class: GITHUB_ACCOUNT_ENTITY_CLASS,
      },
    ],
    relationships: [],
    dependsOn: [],
    executionHandler: fetchAccountDetails,
  },
];
