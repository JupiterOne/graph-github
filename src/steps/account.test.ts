import { Recording } from '@jupiterone/integration-sdk-testing';
import { sanitizeConfig } from '../config';
import { DATA_ACCOUNT_ENTITY, accountSteps } from './account';
import { integrationConfig } from '../../test/config';
import { invocationConfig } from '..';
import { setupGithubRecording } from '../../test/recording';
import { executeStepWithDependencies } from '../../test/executeStepWithDependencies';

let recording: Recording;
afterEach(async () => {
  await recording.stop();
});

test('fetchAccountDetails exec handler', async () => {
  recording = setupGithubRecording({
    directory: __dirname,
    name: 'account', //redaction of headers is in setupGithubRecording
  });
  sanitizeConfig(integrationConfig);
  integrationConfig.installationId = 17214088; //this is the id the recordings are under

  const {
    collectedEntities,
    collectedRelationships,
    encounteredTypes,
    jobState,
  } = await executeStepWithDependencies({
    stepId: accountSteps[0].id,
    invocationConfig: invocationConfig as any,
    instanceConfig: integrationConfig,
  });

  expect({
    numCollectedEntities: collectedEntities.length,
    numCollectedRelationships: collectedRelationships.length,
    collectedEntities: collectedEntities,
    collectedRelationships: collectedRelationships,
    encounteredTypes: encounteredTypes,
  }).toMatchSnapshot();

  const accounts = collectedEntities.filter((e) =>
    e._type.includes('github_account'),
  );

  expect(accounts.length).toEqual(1);
  expect(accounts).toMatchGraphObjectSchema({
    _class: ['Account'],
    schema: {
      additionalProperties: true,
      properties: {
        _type: { const: 'github_account' },
        accountType: { type: 'string' },
        accountId: { type: 'string' },
        login: { type: 'string' },
        _rawData: {
          type: 'array',
          items: { type: 'object' },
        },
      },
      required: ['accountId'],
    },
  });

  const entityFromConstant = await jobState.getData(DATA_ACCOUNT_ENTITY);
  expect(entityFromConstant).toEqual(accounts[0]);
});
