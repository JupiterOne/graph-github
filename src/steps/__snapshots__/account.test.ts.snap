// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchAccountDetails exec handler 1`] = `
Object {
  "collectedEntities": Array [
    Object {
      "_class": Array [
        "Account",
      ],
      "_key": "MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "createdAt": "2021-05-27T15:21:12Z",
            "databaseId": 84923503,
            "description": "Here's my test description",
            "email": "email@email.com",
            "id": "MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz",
            "isVerified": false,
            "location": "Albania",
            "login": "Kei-Institute",
            "name": "Kei-Institute",
            "updatedAt": "2022-01-06T17:48:33Z",
            "url": "https://github.com/Kei-Institute",
            "websiteUrl": "www.goclickatesturl.com",
          },
        },
      ],
      "_type": "github_account",
      "accountId": "Kei-Institute",
      "accountType": "Organization",
      "createdOn": 1622128872000,
      "databaseId": 84923503,
      "description": "Here's my test description",
      "displayName": "Kei-Institute",
      "email": "email@email.com",
      "location": "Albania",
      "login": "Kei-Institute",
      "name": "Kei-Institute",
      "node": "MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz",
      "updatedOn": 1641491313000,
      "verified": false,
      "webLink": "https://github.com/Kei-Institute",
      "websiteUrl": "www.goclickatesturl.com",
    },
  ],
  "collectedRelationships": Array [],
  "encounteredTypes": Array [
    "github_account",
  ],
  "numCollectedEntities": 1,
  "numCollectedRelationships": 0,
}
`;
